From 235601c37ce0dd3c42906d29b20c4f32b15a1b4f Mon Sep 17 00:00:00 2001
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Tue, 20 Aug 2013 15:09:46 +0200
Subject: [PATCH 18/28] pc: set level/xlevel correctly on 486/qemu32 CPU models for rhel6.x

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <1377011392-9336-2-git-send-email-ehabkost@redhat.com>
Patchwork-id: 53609
O-Subject: [RHEL7 PATCH 1/7] pc: set level/xlevel correctly on 486/qemu32 CPU models for rhel6.x
Bugzilla: 918907
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

Bugzilla: 918907
Upstream status: not applicable
  (Upstream have the new xlevel on all machine-types since v0.13)

Upstream commit 58012d66dc7323f48e9bad3be6d65a50ed3d76bc changed xlevel
on the "qemu32" CPU model and level on "486", but we need to keep a
compatible value on the rhel6.x machine-types.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/i386/pc_piix.c | 8 ++++++++
 1 file changed, 8 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/i386/pc_piix.c |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 0f63290..0dc52c4 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -852,6 +852,14 @@ static QEMUMachine pc_machine_rhel700 = {
         .driver   = "AC97",\
         .property = "use_broken_id",\
         .value    = stringify(1),\
+    },{\
+        .driver = "qemu32-" TYPE_X86_CPU,\
+        .property = "xlevel",\
+        .value = stringify(0),\
+    },{\
+        .driver = "486-" TYPE_X86_CPU,\
+        .property = "level",\
+        .value = stringify(0),\
     }
 
 static void pc_init_rhel650(QEMUMachineInitArgs *args)
-- 
1.7.1

